app/models/attachment.rb

```diff
@@ -2,4 +2,18 @@ class Attachment < ActiveRecord::Base
   mount_uploader :file_upload, FileUploader
 
   belongs_to :attachable, polymorphic: true
+
+  # Download url of the file
```

Comment:
These two functions are just wrappers, It's the behavior of carrierwave,  and I think this behavior make sense, the file url will always work in views, because the relative path is relative to the public folder.